frontend/src/components/dashboard/aliases/MaskCard.module.scss

```diff
@@ -142,6 +151,7 @@
     color: $color-grey-40;
     border-radius: $border-radius-sm;
     width: $layout-sm;
+    z-index: 999;
```

Comment:
suggestion: We definitely want to add a comment here and want to stay away from using such a big number for z-index. 

Reason why we want to be mindful with z-index values: 

- Maintainability: Using arbitrary large numbers makes the code harder to understand and maintain. Future US may struggle to understand why a particular z-index value was set so high (most likely me :)).

- Conflicts: If you use a very high z-index value, you might inadvertently cause stacking issues with other elements, especially third-party components that also use z-index - in our case, we use very little third-party CSS (protocol?), but its a good practice.
 
- Debugging: It becomes difficult to debug issues related to stacking context when z-index values are set arbitrarily high. You may end up spending more time figuring out the correct z-index than you would if you had a more organized approach.
 
- Semantics: z-index values are meant to be relative to each other, so it's easier to understand the stacking context when the numbers are closer together and logically organized.
 
- Performance: While not a significant issue, browsers do have to keep track of the stacking context. Using unnecessarily large numbers could, in theory, consume more memory, although this is not going to break our project 🤞🏻 .

I think overall, its important because it makes debugging easier - easier to identify issues related to stacking context specifically. Also, makes it easier for team members to understand the stacking context, which is particularly useful in larger projects like this one.

We should be able to get by with z-index values like 1, 2, 3, etc., and reserve higher values for elements that genuinely need to be above everything else, just not 999+ (hopefully).

Reply:
Thanks for the detailed review! I made a change now.